front/src/app/todo-list/todo-list.component.ts

```diff
@@ -0,0 +1,43 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { ApiService, Lists } from '../shared/services/Api.service';
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-todo-list',
+  templateUrl: './todo-list.component.html',
+  styleUrl: './todo-list.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TodoListComponent implements OnInit{
+
```

Comment:
- add a space between `onInit` and opening bracket
- no need in empty line after brackets opening